install_files/ansible-base/group_vars/all/securedrop

```diff
@@ -51,3 +51,4 @@ appserver_dependencies:
   - redis-server
   - supervisor
 
+tor_apt_repo_url: http://tor-apt.freedom.press
```

Comment:
`apt.freedom.press` supports HTTPS. Why isn't it being used here?

Reply:
so initially there was confusion on my behalf on how to do this with our subdomain and S3 buckets --- but after some much needed time away from that task. I know an excellent way to work around my issue to make it work. So yeah -- i agree, i dont think we should merge until this is https which requires a backend change